src/harness/fake_sas.py

```diff
@@ -211,6 +212,7 @@ def _GetMissingParamResponse(self):
 
 class FakeSasAdmin(sas_interface.SasAdminInterface):
   """Implementation of SAS Admin for Fake SAS."""
+  propagationAntennaModel = QueryPropagationAntennaModel()
```

Comment:
This is a class variable. When using it, we need to make it clear by calling:
FakeSasAdmin.propagationAntennaModel, and not self.PropagationAntennaModel.

But see some of my further comments. The QueryProppagationAntennaModel class has no reason to exists, since it is just a namespace in fact to group function (does not handle any instance variable or method). 
The pythonic way to group related functions is simply to put them in a module, not a void class.
So:
 -  this line should disappear.
 - below when calling  ```self.propagationAntennaModel.computePropagationAntennaModel(request)```, you will have simply to call the static function:  ```util.ComputePropagationAntennaModel(request)```

(see my comments about the nlcddriver and the getPPinPPA function migration).